src/app/(app)/specialist/@modal/(.)[id]/page.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
-import { CardSpecialistExtended } from '@/app/_components/CardSpecialist';
-import { getSpecialistById } from '@/app/(app)/specialist/utils';
+import { CardSpecialistExtended, CardOrganizationExtended } from '@/app/_components/CardSpecialist';
+import { getOrganizationById, getSpecialistById } from '@/app/(app)/specialist/utils';
 
 export default async function Page({ params }) {
   const { id } = params;
   const specialist = await getSpecialistById({ id });
 
-  return <CardSpecialistExtended specialist={specialist} />;
+  if (specialist) {
+    return <CardSpecialistExtended specialist={specialist} />;
+  }
+  const organization = await getOrganizationById({ id });
+  return <CardOrganizationExtended organization={organization} />;
```

Comment:
@Yivgha we should handle the case when there is no organization and no specialist by this id.

Reply:
plain 404 should work i guess 🤔

Reply:
in folder modal we have function Error that hadles this question and shows a layout with image and info

Reply:
changed